parameters/parameters.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2017 Appliscale
+//
+// Maintainers and contributors are listed in README file inside repository.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package parameters provides tools for interactive creation of parameters file for aws
+// cloud formation.
+
+package parameters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/Appliscale/perun/context"
+	"github.com/Appliscale/perun/offlinevalidator"
+	"github.com/Appliscale/perun/offlinevalidator/template"
+	cloudformation2 "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/awslabs/goformation/cloudformation"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+)
+
+type Parameter struct {
+	ParameterKey   string
+	ParameterValue string
+}
+
+func GetJSONParameters(context *context.Context) (resultString []byte, err error) {
+	var parameters []*Parameter
+	parameters, err = GetParameters(context)
+	if err != nil {
+		context.Logger.Error(err.Error())
+		return
+	}
+
+	if *context.CliArguments.PrettyPrint {
+		resultString, err = json.MarshalIndent(parameters, "", "    ")
```

Comment:
Let's move that into common helper in order to have exact same pretty printing used everywhere.